odl/discr/tensor_ops.py

```diff
@@ -669,6 +669,67 @@ def adjoint(self):
                               weight=self.weights)
 
 
+# TODO: Optimize this to an optimized operator on its own.
+class PointwiseSum(PointwiseInner):
+
+    """Take the point-wise sum of a vector field.
+
+    This operator takes the (weighted) sum
+
+        ``sum(F(x)) = [ sum_j( w_j * F_j(x) ) ]
+
+    where ``F`` is a vector field. This implies that
```

Comment:
Vector valued function implies it is actually a function in a stronger sense than vector field does, at least to me. This function accepts anything, i.e. a productspace of rn spaces is fine. So we dont want a very strong notion of function in there.